apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx

```diff
@@ -453,10 +452,8 @@ const orderedDisplayValues = [
   "inline-flex",
   "inline",
   "none",
+  "contents",
 ];
-if (isFeatureEnabled("displayContents")) {
```

Comment:
This is testing only flag. It's not a feature. Should not be removed.

Reply:
got it, putting it back

Reply:
such things need comments